client/modules/accounts/components/auth/loginButtons.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component, PropTypes } from "react";
```

Comment:
This is the case with everything in the project. If you look at the Meteor 1.5 branch there are 150+ linting errors because of it. We can do it here I suppose but it needs to be done everywhere.